scripting/retakes_pistolallocator.sp

```diff
@@ -80,6 +82,8 @@ public void PistolAllocator(ArrayList tPlayers, ArrayList ctPlayers, Bombsite bo
     for (int i = 0; i < tCount; i++) {
         int client = tPlayers.Get(i);
         int choice = g_PistolChoice[client];
+        if(IsRandomPistol(choice)) 
+            choice = GetRandomInt(0, 6);
```

Comment:
Replace 6 with `sizeof(g_PistolChoices) - 2`. Same for the ct loop below.

Reply:
Changed